modules/op-installing-pipelines-as-code-cli.adoc

```diff
@@ -0,0 +1,51 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="installing-pipelines-as-code-cli_{context}"]
```

Comment:
Shouldn't the install section only show the instructions for installing the productised TP CLI?

Reply:
@siamaksade : The upstream documentation https://pipelinesascode.com/docs/guide/cli/#install still shows instructions similar to what I have here. Where do I get the process for productized version?
@sm43 , @chmouel - Please help me out here.